src/osdag/cli_tool.py

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env python
+
+import click
+import os
+import yaml
+
+# @click.command()
+# @click.option('-i', help = 'Enter the input File Path for the report')
+# @click.argument('file_path', type=click.Path(exists=True))
+# def determine_module(file_path):
+#     """
+#     Reads a plain text file line by line, extracts the 'Module' field,
+#     and determines the module type.
+#     """
+#     try:
+#         with open(file_path, 'r') as file:
+#             for line in file:
+#                 # Check if the line contains the 'Module' field
+#                 if line.startswith("Module:"):
+#                     # Extract and print the module name
+#                     module_name = line.split(':', 1)[1].strip()
+#                     click.echo(f"Module: {module_name}")
+#                     return
```

Comment:
I recall in a recent meeting you showed this code as working. Have you not updated the changes here?